scripts/metrics/test_metrics_xml_to_csv.py

```diff
@@ -0,0 +1,198 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Helper file for assisting in building workspaces and setting up LTT with the current Lumberyard environment.
+"""
+import os.path
+import sys
+import csv
+import argparse
+import xml.etree.ElementTree as xmlElementTree
+import datetime
+
+from common import logging
+from common.config import load_config, read_config
+
+CONFIG_FILE = "config/config.yml"
+DEFAULT_CTEST_LOG_FILENAME = 'Test.xml'
+# Format default file name based off of date
+now = datetime.datetime.now()
+MONTH = now.month
+if MONTH < 10:
+    MONTH = f"0{now.month}"
+DAY = now.day
+if DAY < 10:
+    DAY = f"0{now.day}"
+DEFAULT_CSV_FILE = f"{now.year}_{MONTH}_{DAY}.csv"
+
+# Setup logging.
```

Comment:
Logging should not be configured at the top of a script like this, as this will run during import and hijack any existing logging settings of another module that (accidentally) imports this module. Instead, have this called in the main() method or the "__main__" hook.

Reply:
Fixed